glide.yaml

```diff
@@ -29,7 +32,7 @@ import:
   - pkg/types
   - pkg/conversion
 - package: k8s.io/kubernetes
-  version: v1.6.0
+  version: ebb8d6e0fadfc95f3d64ccecc36c8ed2ac9224ef # kube 1.7.0
```

Comment:
I followed the changes done on scheduler for moving to 1.7.  This SHA points to 

commit ebb8d6e0fadfc95f3d64ccecc36c8ed2ac9224ef
Author: Chao Xu <xuchao@silkubi.fra.corp.google.com>
Date:   Fri Jun 30 02:13:41 2017 +0200

    Kubernetes version v1.7.1-beta.0 file updates

diff --git a/api/openapi-spec/swagger.json b/api/openapi-spec/swagger.json
index 8cfb739..3f0dc11 100644
--- a/api/openapi-spec/swagger.json
+++ b/api/openapi-spec/swagger.json
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
    "title": "Kubernetes",
-   "version": "v1.7.0"
+   "version": "v1.7.1"
   },
   "paths": {
    "/api/": {
diff --git a/federation/apis/openapi-spec/swagger.json b/federation/apis/openapi-spec/swagger.json
index 6aca78d..daee7e0 100644
--- a/federation/apis/openapi-spec/swagger.json
+++ b/federation/apis/openapi-spec/swagger.json